texbuild

```diff
@@ -155,17 +150,18 @@ while (True):
 
 	msg ("Built");
 
+	build_path = os.path.join(outputdir, f'{outputfilebasename}.pdf')
 
 	if os.path.exists(final):
 		debug ("copying pdf")
 
 		# Delete before copying for okular
-		if os.path.exists("t.pdf"):
-			os.remove ("t.pdf")
-		shutil.copy (final, "t.pdf")
+		if os.path.exists(build_path):
+			os.remove (build_path)
+		shutil.copy (final, build_path)
 
 	debug ("copying log")
-	shutil.copy (logfile, "t.log")
-	shutil.copy (auxfile, "t.aux") # for bibtex
+	# shutil.copy (logfile, "t.log")
```

Comment:
Does something need to happen here?

Reply:
Oddly enough, no. 

 - The logfile can stay in output/
 - Bibtex works, even if the aux file isn't copied. Not entirely sure why, but ah well

Regardless, I removed the debug statement since that action is no longer conducted.